sdk/src/main/scala/io/horizen/account/api/http/route/AccountMetricsRoute.scala

```diff
@@ -17,18 +17,19 @@ import io.horizen.utils.ClosableResourceHandler
 import io.prometheus.metrics.expositionformats.ExpositionFormats
 import io.prometheus.metrics.model.registry.PrometheusRegistry
 import sparkz.core.api.http.ApiDirectives
-import sparkz.core.settings.RESTApiSettings
+import io.horizen.MetricsApiSettings
 import sparkz.util.SparkzLogging
 import io.horizen.json.Views
 import io.horizen.metrics.{MetricsHelp, MetricsManager}
+import sparkz.core.settings.{ApiSettings, RESTApiSettings}
+
 import java.io.ByteArrayOutputStream
 import scala.concurrent.ExecutionContext
 import scala.reflect.ClassTag
 
 case class AccountMetricsRoute(
-    override val settings: RESTApiSettings,
-    sidechainNodeViewHolderRef: ActorRef,
-    rpcProcessor: RpcProcessor
+      metricsSettings: MetricsApiSettings,
+      sidechainNodeViewHolderRef: ActorRef
 )(implicit val context: ActorRefFactory, override val ec: ExecutionContext)
     extends SidechainApiRoute[
```

Comment:
Does it really need to extends SidechainApiRoute? Maybe it can be directly a subclass of ApiRoute with ApiDirectives, it will simplify a bit.